angular-app/src/app/discovery/discovery.component.html

```diff
@@ -2,38 +2,58 @@
   <span>Discover</span>
 </mat-toolbar>
 
-<mat-selection-list #tags>
-  <mat-list-option *ngFor="let tag of tagOptions">
-    {{tag}}
-  </mat-list-option>
-</mat-selection-list>
+<div class="filter-chip">
+        <mat-chip class="pi-chip-filter"
```

Comment:
This block seems like it is indented too much.

https://google.github.io/styleguide/htmlcssguide.html#Indentation